src/frost.rs

```diff
@@ -389,14 +389,18 @@ pub struct SigningPackage {
     pub message: &'static [u8],
 }
 
+/// A representation of a single signature used in FROST structures and messages.
+#[derive(Clone, Copy, Default)]
+pub struct FrostSignature(Scalar);
+
 /// A participant's signature share, which the coordinator will use to aggregate
 /// with all other signer's shares into the joint signature.
 #[derive(Clone, Copy, Default)]
 pub struct SignatureShare {
     /// Represents the participant index.
     pub(crate) index: u8,
     /// This participant's signature over the message.
-    pub(crate) signature: Scalar,
+    pub(crate) signature: FrostSignature,
```

Comment:
I would call this a `Response`, to be more specific

Reply:
@chelseakomlo what about something related to the fact that this is share that contributes to the `s` in the Schnorr signature? `frost::SignatureShareS`, and maybe as an alias for now. @teor2345

Reply:
I don-t think we should block on the name, we can pick any, then do search and replace if we need to change it.

Reply:
I think for now `SignatureResponse` works

Reply:
Awesome :)